segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/GarbageCollectorTests.java

```diff
@@ -36,21 +41,24 @@
 
 import java.io.ByteArrayInputStream;
 import java.time.Duration;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashSet;
+import java.util.TreeMap;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.function.Supplier;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
 /**
  * Tests for {@link GarbageCollector}.
  */
 @Slf4j
 public class GarbageCollectorTests extends ThreadPooledTestSuite {
     public static final int CONTAINER_ID = 42;
+    public static final long TXN_ID = 123;
     protected static final Duration TIMEOUT = Duration.ofSeconds(3000);
```

Comment:
This is pretty large. Can we set it to 30 seconds please.

Reply:
oops.